client/src/Dashboard/DashboardTable.js

```diff
@@ -30,11 +30,15 @@ export default function DashboardTable({
   const dispatch = useDispatch()
   const { sendGAEvent } = useGoogleAnalytics()
   const [isMIModalVisible, setIsMIModalVisible] = useState(false)
+  const [isAUModalVisible, setIsAUModalVisible] = useState(false)
   const [selectedChild, setSelectedChild] = useState({})
+  const [effectiveDate, setEffectiveDate] = useState(null)
+  const [expirationDate, setExpirationDate] = useState(null)
   const [inactiveDate, setInactiveDate] = useState(null)
   const [activeDate, setActiveDate] = useState(null)
   const [inactiveReason, setInactiveReason] = useState(null)
   const [sortedRows, setSortedRows] = useState([])
+  const [auErrorMessage, setAUErrorMessage] = useState('')
```

Comment:
is this a typo? setAUE (AU in capital letters)

Reply:
Is meant for Authorization Update, maybe changing auErrorMessage to aUErrorMessage would be more consistent?

Reply:
yes, that makes sense